electrumx/lib/coins.py

```diff
@@ -369,6 +372,58 @@ def find_end_position_of_name(script, length):
 
         return n
 
+    @classmethod
+    def interpret_name_prefix(cls, script, possible_ops):
+        """Interprets a potential name prefix
+
+        Checks if the given script has a name prefix.  If it has, the
+        name prefix is split off the actual address script, and its parsed
+        fields (e.g. the name) returned.
+
+        possible_ops must be an array of arrays, defining the structures
+        of name prefixes to look out for.  Each array can consist of
+        actual opcodes, -1 for ignored data placeholders and strings for
+        named placeholders.  Whenever a data push matches a named placeholder,
+        the corresponding value is put into a dictionary the placeholder name
+        as key, and the dictionary of matches is returned."""
+
+        from electrumx.lib.script import _match_ops, Script, ScriptError
```

Comment:
This might be better added to the module-level import of the same?

Reply:
I just kept this how it was in the code I refactored, but I agree that it makes sense to move that to the module level.  Done.